docs/content/quickstart.md

```diff
@@ -3,116 +3,127 @@ title: QuickStart Guide
 descriptions: Get started using Porter
 ---
 
+So you're interested in learning more about Porter? Great! This guide will walk you through key concepts of managing bundles. You will use the porter CLI to install, upgrade, and uninstall the bundle. 
+
+## Pre-requisites
+
+Docker is currently a prerequisite for using Porter. Docker is used to package up the bundle. 
+
+If you do not have Docker installed, go ahead and [get Docker](https://docs.docker.com/get-docker/). 
+
 ## Getting Porter
 
-First make sure Porter is installed.
-Please see the [installation instructions](/install/) for more info.
+Next, you need Porter. Follow the Porter [installation instructions](/install/).
 
-## Create a new bundle
-Use the `porter create` command to start a new project:
+## Porter Key Concepts 
 
-```
-mkdir -p my-bundle/ && cd my-bundle/
-porter create
-```
+For this quickstart, the main concepts that you will use include:
 
-This will create a file called **porter.yaml** which contains the configuration
-for your bundle. Modify and customize this file for your application's needs.
+* Bundle - A bundle is your application artifact, client tools, configuration and deployment logic packaged together. 
+* Installation - An instance of a bundle installed to your system.
+* Tag - The metadata that contains information about the registry, image name, and version. 
+* Registry - An OCI compliant artifact store that allows machine image management. 
 
-Here is a very basic **porter.yaml** file:
+## Inspect a bundle
```

Comment:
We should avoid using "inspect" here since that's a different command. I usually say "explain how to use a bundle"